R/process_tests_sc_ch_episodes.R

```diff
@@ -0,0 +1,17 @@
+#' Process sc ch episodes tests
+#'
+#' @description This script takes the processed demographic extract and produces
+#' a test comparison with the previous data. This is written to disk as a csv.
+#'
+#' @return a csv document containing tests
+#' @export
+#'
+process_tests_sc_ch_episodes <- function() {
+  comparison <- produce_test_comparison(
+    old_data = produce_sc_ch_episodes_tests(readr::read_rds(get_sc_ch_episodes_path(update = previous_update()))),
+    new_data = produce_sc_ch_episodes_tests(readr::read_rds(get_sc_ch_episodes_path()))
+  ) %>%
+    write_tests_xlsx(sheet_name = "all_ch_episodes")
+
+  return(comparison)
```

Comment:
To check: I think `comparison` will be the path to the Excel workbook (because that's what `write_tests_xlsx()` returns).

Options are:

1. Leave it as it is (but change the documentation slightly so that the path to the workbook will be returned.
2. Return the comparison from this function.
```suggestion
  comparison <- produce_test_comparison(
    old_data = produce_sc_ch_episodes_tests(readr::read_rds(get_sc_ch_episodes_path(update = previous_update()))),
    new_data = produce_sc_ch_episodes_tests(readr::read_rds(get_sc_ch_episodes_path()))
  )
  
  write_tests_xlsx(comparison, sheet_name = "all_ch_episodes")

  return(comparison)
```
3. Or modify `write_tests_xlsx()` to return the comparison (i.e. the actual data / tibble), which would mean this function doesn't need changing.

I think I prefer option 3 but it's obviously the most far-reaching change that would affect other bits of code, possibly that's fine though.

Reply:
Had a call with @Moohan and made these changes to other sc functions